config/initializers/assets.rb

```diff
@@ -12,3 +12,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w(accessible-autocomplete/src/autocomplete.css)
```

Comment:
Should this be `accessible-autocomplete/dist/accessible-autocomplete.min.css`?

Reply:
Sorry, this is a draft, just wanted to try it in a review app on heroku!

Reply:
Yep, but as the assets are compiled anyway the src makes more sense to me and to be honest, I cannot get the asset pipeline to use the minified version!